tests/unit/test_termio.py

```diff
@@ -30,3 +36,20 @@ def test_term_interactive(ps1, force_flag, expect, monkeypatch):
         monkeypatch.delitem(os.environ, "GLOBUS_CLI_INTERACTIVE", raising=False)
 
     assert term_is_interactive() == expect
+
+
+def test_format_record_list(capsys):
+    data = [
+        {"bird": "Killdeer", "wingspan": 46},
+        {"bird": "Franklin's Gull", "wingspan": 91},
+    ]
+    fields = [("Bird", "bird"), ("Wingspan", "wingspan")]
+    with click.Context(click.Command("fake-command")) as _:
+        formatted_print(data, text_format=FORMAT_TEXT_RECORD_LIST, fields=fields)
```

Comment:
I'm perfectly happy with this, if you want to keep it, but I'll point out some other tests we have which define a click command _inline_ in order to check behavior:
https://github.com/globus/globus-cli/blob/7db15fbb09d3bb3f6e6aa22f04520936af0b99cf/tests/unit/test_param_types.py#L24

If you prefer that, feel free to switch.

Otherwise, I wonder if `with click.Command("foo").make_context():` reads a bit better.

Reply:
Yeah, I felt like this was a little more clear since we're not concerned with any command setup, just what gets printed.

Reply:
Sounds good to me. Marking resolved.